utils/common.sh

```diff
@@ -106,3 +106,83 @@ gen_spreadsheet() {
     rm -rf ${csv_tmp}
   fi
 }
+
+
+
+##############################################################################
+# Creates a new document containing cluster information
+# Arguments:
+#   Benchmark name (as `oc get benchmark` displays it
+#   start_date (epoch)
+#   end_date (epoch)
+##############################################################################
+
+gen_metadata() {
+        BENCHMARK=$1
+        START_DATE=$2
+        END_DATE=$3
+
+        # construct all the required information
+        local VERSION_INFO=$(oc version -o json)
+        local INFRA_INFO=$(oc get infrastructure.config.openshift.io cluster -o json)
+        local PLATFORM=$(echo ${INFRA_INFO} | jq -r .spec.platformSpec.type)
```

Comment:
As I have mentioned on https://github.com/cloud-bulldozer/e2e-benchmarking/pull/332#pullrequestreview-902739288, platform type is not enough to differentiate AWS clusters from ROSA ones.

We need to check the tags included on platformStatus:

```
       "platformStatus": {
            "aws": {
                "region": "us-west-2",
                "resourceTags": [
                    {
                        "key": "red-hat-clustertype",
                        "value": "rosa"
                    },
                    {
                        "key": "red-hat-managed",
                        "value": "true"
                    }
                ]
            },
            "type": "AWS"
        }
    }
```

Reply:
Checking if resourceTags[0].value is rosa now and adding it as CLUSTERTYPE to the document, let me know if that's enough

Reply:
yep, now it is perfect